apiserver/pkg/manager/resource_manager.go

```diff
@@ -30,6 +30,10 @@ type ResourceManagerInterface interface {
 	GetComputeTemplate(ctx context.Context, name string, namespace string) (*v1.ConfigMap, error)
 	ListComputeTemplates(ctx context.Context, namespace string) ([]*v1.ConfigMap, error)
 	DeleteComputeTemplate(ctx context.Context, name string, namespace string) error
+	CreateJob(ctx context.Context, apiCluster *api.Cluster) (*v1alpha1.RayJob, error)
+	GetJob(ctx context.Context, jobName string, namespace string) (*v1alpha1.RayJob, error)
+	ListJobs(ctx context.Context, namespace string) ([]*v1alpha1.RayJob, error)
```

Comment:
should we add `DeleteJob` as well?

Reply:
added.
BTW, should we use ResourceManagerInterface in each component?